cmd/virt-handler/virt-handler.go

```diff
@@ -312,6 +315,20 @@ func (app *virtHandlerApp) Run() {
 		}
 	}
 
+	if app.MaxDevices == 0 {
+		yamlFile, err := ioutil.ReadFile(kubeletConfigDir)
+		if err != nil {
+			glog.Fatalf("Error reading kubelet config: %v", err)
+		}
+		var data map[string]interface{}
+		err = yaml.Unmarshal(yamlFile, &data)
+		if err != nil {
+			glog.Fatalf("Error unmarshalling kubelet config: %v", err)
+		}
+		app.MaxDevices = data["maxPods"].(int)
```

Comment:
I don't think this value is required or defaulted in the configuration file. We should set a default if this is not present.